autocertkit/utils.py

```diff
@@ -65,6 +65,7 @@
 
 # Capability Tags
 REQ_CAP = "REQ"
+REQ_CAP_MULTICAST = "REQ_MULTICAST"
```

Comment:
@fillzero ,

I see self.tags doesn't have a complete implementation. 
I am ok with above change except name  REQ_MULTICAST which implies as "REQuired capability MULTICAST" . Can you rename it to just say MULTICAST or better something like VM_MULTICAST